app/models/manageiq/providers/openstack/storage_manager/cinder_manager/cloud_volume.rb

```diff
@@ -55,13 +55,73 @@ def self.params_for_create(ems)
             }
           end,
         },
+        {
+          :component    => 'select',
+          :name         => 'volume_type1',
```

Comment:
Why `volume_type1` not `volume_type` ?  Does `params_for_create` clash with `params_for_update` somehow?

Reply:
I was using that for testing, just forgot to remove it. Will fix and repost.